src/david/feature.js

```diff
@@ -1 +1,92 @@
- console.log("david");
\ No newline at end of file
+ console.log("david");
+
+/*
+14= number
+-80= number
+8.5 = number
+true -- boolean
+(5 < 17) -- boolean
+null undefined
+3.142 -- number
+"Pi"-- string
+67+18 = number
+*/
+
+//convert number to base 2 
+var base2converter = function(number) {
+    var i= parseInt(number, 10).toString(2);
+    console.log(i);
```

Comment:
It's recommended that you redefine your functions to use the `return` keyword instead of the `console.log` method.